src/include/serial_inline.h

```diff
@@ -299,11 +299,6 @@ __wt_update_serial(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_PAGE *page
     if (upd->next == NULL || exclusive)
         return (0);
 
-    /*
-     * We would like to call __wt_txn_update_oldest only in the event that there are further updates
-     * to this page, the check against WT_TXN_NONE is used as an indicator of there being further
-     * updates on this page.
-     */
```

Comment:
In the early measurements I did when playing with this change - a large part of the cost was due to repeatedly updating the oldest via the call below to wt_txn_update_oldest. Would it make sense to wrap this into the code that limits how often obsolete content is looked for?